### ISSUE TITLE
Easier "vault add"

### ISSUE THREAD

**Issue by andresriancho** [franela/vault#19]
The `vault add` command could be improved by letting the user run something like:

`vault add foo@bar.com`

Instead of forcing the user to search for the signature. Of course this will only work if the user already trusts the key associated with foo@bar.com

**Comment by xetorthio** [comment]
I don't really understand how to do this.
Because I could have in my key ring different keys with the same email associated to them. Which should I add?

**Comment by marcosnils** [comment]
@xetorthio that's correct. Initially we wanted vault to be like this, but we found that emails can be duplicated in the keyring. That's why we used the fingerprint as a unique identifier.

**Comment by xetorthio** [comment]
What we could do is to allow it, but then when there is more than 1 key with an e-mail warn the user and exit.
But in the case you don't have repeated e-mails (which I think could be the most common case) `vault add <email>` could work and be a lot nicer :)
